tokio/src/runtime/task/join.rs

```diff
@@ -296,6 +296,7 @@ impl<T> JoinHandle<T> {
     /// # }
     /// ```
     /// [cancelled]: method@super::error::JoinError::is_cancelled
+    #[must_use]
```

Comment:
You can add a warning for the user to provide more information.
```suggestion
    #[must_use = "your warning"]
```

Reply:
Added